### PR TITLE
add contact section

### PR DESCRIPTION
**Pull request by AbbyNyakara**
Added the contact form in the mobile webpage

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -281,3 +281,75 @@ button {
...
+.contact-form form > input {
+  height: 40px;
+  border: none;
+  border-radius: 5px;
+}
+
+.contact-form form > input::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
```
Comment (OmarRamoun) [inline style.css:335]: - [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

image

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.contact-form form textarea::placeholder {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}

.contact-form form > input::placeholder {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: inter, sans-serif;
}
```

> **Note**: Also, you can do the same thing with the `display: flex` declaration.

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @AbbyNyakara 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ descriptive commit messages
- ✔️ descriptive PR title and description
- ✔️🎯 Great UI (Following the figma design very good)
- The code is top-notch! 💯 and has a huge potential.

> **Reviewer Note:** I like how the student is following the Figma design and sticking to it in a very great way 💯 👍 (Good Job)

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code_

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AbbyNyakara) [reply style.css:335]: Thanks. This is actually genius. i'll apply the same class instead!

Comment (AbbyNyakara) [reply style.css:335]: @MrRamoun that font is for the placeholder. When i add it as a class to the input and textarea tags, there is actually no effect. What suggestion do you have?

Comment (OmarRamoun) [reply style.css:335]: @AbbyNyakara you can use id instead of class. I believe it is a specificity issue

**Pull request merged**